### PR TITLE
Suppress Calcite CVE

### PR DESCRIPTION
**Pull request by adarshsanjeev**
The vulnerability is in a builtin function in Calcite, EXTRACT_VALUE, which is not exposed in the Druid console, so druid should be unaffected.

- Suppress calcite CVE.

<hr>

This PR has:
- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: owasp-dependency-check-suppressions.xml [hunk 1]
```diff
@@ -632,4 +632,26 @@
     <cve>CVE-2022-31197</cve>
   </suppress>
 
+  <suppress>
+    <!-- avatica-server-1.17.0.jar -->
+    <notes><![CDATA[
+   file name: avatica-server-1.17.0.jar
```
Comment (kfaraz) [inline owasp-dependency-check-suppressions.xml:638]: The vulnerability is in calcite-core. Does it need to be suppressed for avatica-server too?

Comment (adarshsanjeev) [reply owasp-dependency-check-suppressions.xml:638]: I believe calcite is used by avatica server too, wouldn't it need to be suppressed separately?

Comment (kfaraz) [reply owasp-dependency-check-suppressions.xml:638]: I don't think so. Could you try running the vulnerability check without the extra suppression?

File: owasp-dependency-check-suppressions.xml [hunk 2]
```diff
@@ -632,4 +632,26 @@
...
+    <cve>CVE-2022-39135</cve>
+  </suppress>
+
+  <suppress>
+    <!-- calcite-core-1.21.0.jar -->
+    <notes><![CDATA[
+   file name: calcite-core-1.21.0.jar
+   ]]></notes>
+    <!--
+    We do not expose any of the built-in functions related to this CVE
```
Comment (kfaraz) [inline owasp-dependency-check-suppressions.xml:652]:
```suggestion
    We do not expose any of the SQL operators that were found vulnerable in this CVE.
```

Comment (adarshsanjeev) [reply owasp-dependency-check-suppressions.xml]: Changed

File: owasp-dependency-check-suppressions.xml (hunk 1 above)
Comment (abhishekagarwal87) [reply owasp-dependency-check-suppressions.xml:638]: the failure is being reported for `avatica-server` as well. 
```
[ERROR] ----------------------------------------------------
[ERROR] .NET Assembly Analyzer could not be initialized and at least one 'exe' or 'dll' was scanned. The 'dotnet' executable could not be found on the path; either disable the Assembly Analyzer or add the path to dotnet core in the configuration.
[ERROR] ----------------------------------------------------
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.0.4:aggregate (default-cli) on project druid: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
[ERROR] 
[ERROR] avatica-server-1.17.0.jar: CVE-2022-39135(9.8)
[ERROR] calcite-core-1.21.0.jar: CVE-2022-39135(9.8)
[ERROR] 
[ERROR] See the dependency-check report for more details.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Review by abhishekagarwal87 — APPROVED** [review]

**Pull request merged**